### PR TITLE
Add additional tags

### PR DESCRIPTION
**Pull request by Mr-Pi**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -18,4 +18,8 @@
     - ssh
     - ssh_moduli
 - import_tasks: hostname.yml
+  tags:
+    - hostname
 - import_tasks: locale.yml
+  tags:
+    - localisation
```
Comment (SlothOfAnarchy) [inline tasks/main.yml]: *localization

File: tasks/users.yml [hunk 2]
```diff
@@ -53,3 +53,5 @@
     owner: root
     group: shadow
     state: present
+  tags:
+    - root_user
```
Comment (SlothOfAnarchy) [inline tasks/users.yml]: Why do you need a tag specifically for the root user? Do you not want to execute this task?

Comment (Mr-Pi) [reply tasks/users.yml]: Indeed, it should be possible to keep the root user active, maybe it's nicer to add a var named deny_root_login and check for it's value?

**Comment by Mr-Pi** [comment]
please rereview

File: tasks/users.yml (hunk 2 above)
Comment (SlothOfAnarchy) [reply tasks/users.yml]: No, I'm against this, why would we need the root user?

Comment (Mr-Pi) [reply tasks/users.yml]: It's the default to remove the root user, but there should be an option to keep them. The default is and should be to disable root logins.

Comment (SlothOfAnarchy) [reply tasks/users.yml]: The whole security effict oft this task is gone if we allow to disable this. On what machine do you need a root user?

**Comment by SlothOfAnarchy** [comment]
So can we close this @Mr-Pi ?

**Pull request closed without merging**